<launch>
	<node pkg="tissue_waypoints" type ="tissue_waypoints" name="tissue_waypoints" output = "screen">
		<param name ="x_min" value ="0.00"/>
		<param name ="x_max" value ="0.04"/>
		<param name ="y_min" value ="-0.01"/>
		<param name ="y_max" value ="0.025"/>
		<param name ="z_min" value ="0.4"/>
		<param name ="z_max" value ="0.5"/>
	</node>
</launch>
